### PR TITLE
fix some tests

### PR DESCRIPTION
**Pull request by enricosada**
some fix and cleanup from #90

### CODE REVIEW

File: tests/fsharp/core/measures/test.fsx [hunk 1]
```diff
@@ -509,7 +509,8 @@ module MembersTest =
     let s = 2.0f<kg>
     let d = 2.0M<kg>
 
-#if !NetCore
+#if NetCore
+#else
```
Comment (latkin) [inline tests/fsharp/core/measures/test.fsx]: Why this change? We finally have decent preprocessor, let's use it :-)

**Comment by latkin** [comment]
LGTM

**Comment by latkin** [comment]
@enricosada failure of diamondAssembly was silly test issue, I fixed that too.  Thanks for the cleanup!

File: tests/fsharp/core/measures/test.fsx (hunk 1 above)
Comment (enricosada) [inline tests/fsharp/core/measures/test.fsx]: i got this time ago during the nunit testsuite pr, running test with old version. 
Maybe now is ok, sry for late response
